### DISCUSSION TITLE
Lab 1 Part 1 Feedback

### DISCUSSION THREAD

**Discussion by shivanidoshi26** [snlee159/Interactive-Lab-Hub#6] [General]
- Maybe the buttons could be more conveniently placed
- The brighter colors should flash when the Study Buddy has exhausted itself
- Maybe instead of just light there could be some sort of vibration and the device could be more of a wearable (so it maintains the promise of not disturbing others)

**Comment by singhaniasnigdha** [comment]
- I like the thought behind the product - this could be so useful in a library setting, or even for people with hearing disabilities!

- If I’m really caught up with working on something, I am likely to completely ignore the color changes. Curious to see how you can expand on this in the next iteration to draw the student’s attention :smile:

**Comment by anam884** [comment]
I really like the idea and the rationale behind it also loved the costume :grin: The context was clear, however, I was confused about how is the timer set? How does the user define when should the clock go off? The reset button gives me a hint that it’s programmed for one fixed time duration, is that true?

Something you might wanna consider for the next iteration:
Will it always be used indoors? How about if someone uses it while studying outside or if their table is by the window. Will the change of colors be prominent? Maybe you can add vibration in order to keep it in the silent category.
